### PR TITLE
Buc 1525 improve feedback documentation

### PR DESCRIPTION
**Pull request by Munter**

### CODE REVIEW

**Review by roncohen — APPROVED** [review]
Nice Peter! would be nice to show a real example of translated message keys. All the examples have empty `translations: {` lines.

File: FEEDBACK.md [hunk 1]
```diff
@@ -36,75 +38,33 @@ See also:
...
+The live connection for automated feedback is established once you have identified a user with `bucket.user()`.
+
+### Disabling automated feedback collection
 
-You can enable automated collection in the `bucket.init()`-call:
+You can disable automated collection in the `bucket.init()`-call:
 
 ```javascript
-bucke.init({
+bucke.init("bucket-tracking-key", {
```
[the remark this answers is not in the document]
Comment (Munter) [reply FEEDBACK.md:54]: Nice catch

File: FEEDBACK.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 The Bucket SDK includes a UI you can use to collect feedback from user about particular features.
 
+![image](https://github.com/bucketco/bucket-tracking-sdk/assets/331790/7172a053-5ff4-4f1a-b1da-e2ff0ef9770e)
```
[the remark this answers is not in the document]
Comment (Munter) [reply FEEDBACK.md]: Fixed

**Comment by roncohen** [comment]
nice! my french isn't good enough to judge those, but great to have
the example in!

**Pull request merged**